pages/mesosphere/dcos/1.14/deploying-services/creating-services/health-checks/index.md

```diff
@@ -75,5 +75,18 @@ And here is the same health check specified by using the DC/OS UI.
 
 Figure 1. Web interface health check
 
+## Command health check performance
+
+When deploying a service which makes use of command health checks, the resources consumed by the health check itself must be considered. If the command executed by the health check requires significant CPU or memory to run, those resources should be added to the resource requirements of the task. In cases where a task only specifies a very small amount of CPU (0.1 or less), even a trivial health check will have trouble running reliably, so CPU greater than this should be specified on any task making use of a command health check.
```

Comment:
for "so CPU greater than this..." can we make some recommendations? say "1 cpu"?